lib/client.js

```diff
@@ -1,5 +1,5 @@
 const uuid = require('uuid')
-const esClient = require('../src/client')
+const core = require('./dist')
```

Comment:
I've got mixed feelings about requiring from `dist.js` build file in the source file. Shouldn't the source files be based entirely on other source files? 

I guess it has something to do with `lib` directory being separate from `src`, but I find it a bit confusing.

Thinking about it further, it looks like all the files in the `lib` directory are actually the "resulting" package files, but only the `dist.js` is actually built from source and `client.js` is "built" manually and committed to the repo and doesn't have it's "source" files in `src`. 

Not forcing any change here, just let me know if my thinking is correct please :)